src/ape/managers/config.py

```diff
@@ -192,7 +192,8 @@ def _plugin_configs(self) -> Dict[str, PluginConfig]:
         )
 
         for plugin_name, config_class in self.plugin_manager.config_class:
-            user_override = user_config.pop(plugin_name, {})
+            # `or {}` to handle the case when the empty config is `None`.
```

Comment:
for some reason this doesnt happen to all configs, which is maybe why @z80dev couldnt repro, but i get it specifically on the hardat plugin, and i have verified this fixes it.